README.rst

```diff
@@ -0,0 +1,76 @@
+Example: Basic Sphinx project for Read the Docs
+===============================================
+
+This example shows the an integration of a basic Sphinx project with Read the Docs. You're encouraged to view it to get inspiration and copy & paste from the files in the source code. If you are using Read the Docs for the first time, have a look at the official `Read the Docs Tutorial <https://docs.readthedocs.io/en/stable/tutorial/index.html>`_.
+
+📚 `docs/ <https://github.com/readthedocs-examples/example-sphinx-basic/blob/main/docs/>`_
+    A basic Sphinx project lives in ``docs/``, it was generated using Sphinx defaults
```

Comment:
What are the sphinx defaults? `sphinx-quickstart`?

Reply:
Yes! Isn't that safe to call defaults?

Reply:
Marking as resolved for now - LMK if you would want a different phrasing @ericholscher

Reply:
I think we need to be explicit. If a user doesn't know what sphinx defaults are, how do they learn?